ansible-scylla-node/defaults/main.yml

```diff
@@ -274,6 +274,23 @@ scylla_ssl:
   client:
     enabled: true
 
+# Local keys are used for encrypting user data, such as SSTables.
```

Comment:
The patch subject is confusing: you need to be more specific about which keys exactly you are managing here. And this patch manages "Local Keys"